ovos_plugin_manager/templates/tts.py

```diff
@@ -154,59 +93,26 @@ def __init__(self, lang="en-us", config=None, validator=None,
         self.stopwatch = Stopwatch()
         self.tts_name = self.__class__.__name__
         self.bus = BUS()  # initialized in "init" step
```

Comment:
The comment here doesn't make sense to me. `BUS()` is an instance of `FakeBus()` but the init step doesn't allow an override. Clarifying that in a docstring or in this comment would make it easier for a developer to extend without having intimate familiarity with OVOS.

Reply:
yeah this one is confusing, but in essence mycroft required TTS plugins to have a bus attached, in OVOS we dont require that if using plugins standalone, but we need to keep the self.bus property valid so the FakeBus is there as a placeholder just to avoid syntax errors

the init method is called by ovos-audio and assigns a bus and playback thread, without calling init  (only ovos-audio does that) the TTS is used standalone and only to create the .wav files

but yeah, that should probably be None instead of a FakeBus

Reply:
Any objection to making this available from this base class's `__init__` method rather than forcing an override? I'm also still in favor of making this information more explicit to developers

Reply:
not sure what you mean, we cant change the public facing api, the TTS class should not have a bus object at all, i dont want to add it in the init method, im just unable to remove it fully because of compat... 

a bus should only be passed in the `init` (no underscores) method which connects the TTS to the playback queue

Reply:
did you miss the other init method?

```
def init(self, bus=None, playback=None):
        """ Performs intial setup of TTS object.
        Arguments:
            bus:    OpenVoiceOS messagebus connection
```